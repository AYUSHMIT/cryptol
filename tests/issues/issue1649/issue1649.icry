:l Test.cry
B::b ()