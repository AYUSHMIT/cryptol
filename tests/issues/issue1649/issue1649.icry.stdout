Loading module Cryptol
Loading module Cryptol
Loading interface module BInterface
Loading module BFunctor
Loading module A
Loading module Test
()
